apis/core/bootstrap/cubes.ts

```diff
@@ -0,0 +1,13 @@
+import { turtle } from '@tpluscode/rdf-string'
+import { hydra, rdf } from '@tpluscode/rdf-ns-builders'
+import { cube } from '@cube-creator/core/namespace'
+
+export const cubes = turtle`
+<cubes> {
+  <cubes> a ${hydra.Collection} ;
+    ${hydra.manages} [
+      ${hydra.property} ${rdf.type} ;
+      ${hydra.object} ${cube.Cube}
+    ] .
```

Comment:
Hydra crash course 💥 

`hydra:manages` declares indirect statements about the collection contents. There are always two properties from `subject/predicate/object` describing a triple pattern.

The example above would translate to something like 

```SPARQL
SELECT ?member 
{ 
  ?member 
    rdf:type  # hydra:property
    cube:Cube # hydra:object
}
```

And accurate enough, `@hydrofoil/labyrinth` does something similar as a basis for a query to retrieve the `/cubes` resource